generator/generator_service.go

```diff
@@ -129,6 +129,12 @@ func buildInputTypeAndOutputType(parametersTypeName string, responseTypeName str
 	if responseTypeName == "" {
 		outputType = "google.protobuf.Empty"
 	}
+	for _, t := range types {
```

Comment:
Does this change affect the integration tests?

Reply:
https://github.com/google/gnostic-grpc/blob/main/generator/renderer_test.go

Reply:
@LorenzHW all of the tests that were originally passing are still passing. I've fixed the GH action [here](https://github.com/google/gnostic-grpc/pull/91).

I think this test failure:
> 2023/06/06 10:23:55 The query parameter with the Name param2 is invalid. Note that fields which are mapped to URL query parameters must have a primitive type or a repeated primitive type or a non-repeated message type. See: https://github.com/googleapis/googleapis/blob/master/google/api/http.proto#L118 for more information.

Has always been an issue, based on the TODO in parameters.yaml.

Reply:
Thanks @TylerGillson. I think at some point the tests were green but it is ok for now. Still haven't had the chance to take a look at the repository but it is high on my TODO list...

Reply:
No problem, happy to help.